.github/workflows/mutexbot.yml

```diff
@@ -0,0 +1,61 @@
+name: MutexBot Container Image
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "mutexbot/**"
+  push:
+    branches:
+      - main
+    paths:
+      - "mutexbot/**"
+    tags:
+      - "mutexbot-v*.*.*"
+
+permissions:
+  contents: read
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
```

Comment:
Let's use self-hosted native (i.e. without cross-compilation) runners for that

Reply:
I wasn't sure this was worth the extra complexity, given that I do not expect this to run that often, but after <3 minutes compared to >30 minutes is quite a difference.